code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -7265,17 +7264,32 @@ public PlayerCharacter clone()
 		}
 
 		aClone.levelInfoFacet.removeAll(aClone.id);
-		for (PCLevelInfo info : getLevelInfo())
+		try
 		{
-			PCLevelInfo newLvlInfo = info.clone();
-			aClone.levelInfoFacet.add(aClone.id, newLvlInfo);
+			for (PCLevelInfo info : getLevelInfo())
+			{
+				PCLevelInfo newLvlInfo = info.clone();
+				aClone.levelInfoFacet.add(aClone.id, newLvlInfo);
+			}
+		}
+		catch (CloneNotSupportedException e)
```

Comment:
Since this is a checked exception and we'd need to add this to the signature for a long length. Another option is to catch and throw a different exception.